packages/mds-geography/api.ts

```diff
@@ -0,0 +1,170 @@
+        }
+        return res.status(200).send(geography)
+      } catch (err) {
+        await log.error('failed to read geography', err.stack)
+        if (err instanceof NotFoundError) {
+          return res.status(404).send({ error: err })
+        }
+
+        if (err instanceof InsufficientPermissionsError) {
+          return res.status(403).send({ error: err })
+        }
+
+        return res.status(500).send({ error: new ServerError() })
+      }
+    }
+  )
+
+  app.get(
+    pathsFor('/geographies'),
+    checkAccess(scopes => {
+      return scopes.includes('geographies:read:published') || scopes.includes('geographies:read:unpublished')
+    }),
+    async (req, res) => {
+      const summary = req.query.summary === 'true'
+      const { get_published = null, get_unpublished = null } = req.query
+      const params = { get_published, get_unpublished }
+      if (get_published) {
+        params.get_published = get_published === 'true'
+      }
+
+      if (get_unpublished) {
+        params.get_unpublished = get_unpublished === 'true'
+      }
+
+      try {
+        if (!res.locals.scopes.includes('geographies:read:unpublished') && params.get_unpublished) {
+          throw new InsufficientPermissionsError(
+            'Cannot require unpublished geos without geography:read:unpublished scope'
+          )
+        }
+
+        const geographies = summary ? await db.readGeographySummaries(params) : await db.readGeographies(params)
+        if (!res.locals.scopes.includes('geographies:read:unpublished')) {
+          const filteredGeos = geographies.filter(geo => !!geo.publish_date)
+          return res.status(200).send(filteredGeos)
+        }
+        return res.status(200).send(geographies)
+      } catch (error) {
+        /* We don't throw a NotFoundError if the number of results is zero, so the error handling
+         * doesn't need to consider it here.
+         */
+        if (error instanceof InsufficientPermissionsError) {
+          return res.status(403).send({ error })
+        }
+        await log.error('failed to read geographies', error.stack)
+        return res.status(500).send({ error: new ServerError() })
+      }
+    }
+  )
+
+  app.get(
+    pathsFor('/geographies/:geography_id/meta'),
+    checkAccess(scopes => {
+      return scopes.includes('geographies:read:published') || scopes.includes('geographies:read:unpublished')
+    }),
+    async (req, res) => {
+      const { geography_id } = req.params
+      try {
+        const geography_metadata = await db.readSingleGeographyMetadata(geography_id)
+        const geography = await db.readSingleGeography(geography_id)
+        if (!geography.publish_date && !res.locals.scopes.includes('geographies:read:unpublished')) {
+          throw new InsufficientPermissionsError('permission to read metadata of unpublished geographies missing')
+        }
+        return res.status(200).send(geography_metadata)
+      } catch (err) {
```

Comment:
If you just call this `error` then you can do `res.status(xxx).send(error)` in some of the cases rather than having to do `res.status(xxx).send(error: err)`. Applies in a few other places as well